src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -77,6 +77,7 @@ static constexpr std::string_view MoveFocusRightKey{ "moveFocusRight" }; // Lega
 static constexpr std::string_view MoveFocusUpKey{ "moveFocusUp" }; // Legacy
 static constexpr std::string_view MoveFocusDownKey{ "moveFocusDown" }; // Legacy
 static constexpr std::string_view ToggleFullscreenKey{ "toggleFullscreen" };
+static constexpr std::string_view SplitPaneKey{ "splitPane" };

```

Comment:
nit: personally prefer that `splitPane` be next to `splitHorizontal` and `splitVertical`.